### ISSUE TITLE
Page API is broken, can't create submenu pages.

### ISSUE THREAD

**Issue by Neidra** [themosis/framework#310]
Hey,
I tried to create a sub (administration) page.
So I created two pages with some fields. I can access of them in my administration panel. 
```
$page1 = Page::make('parent', 'Parent')->set();
$page1-> ....[creation of fields section and other]
$page2 = Page::make('blip', 'Blip')->set();
$page2-> ....[creation of fields section and other]
```
but when i just add the parent parameter, that doesn't work
```
$page1 = Page::make('parent', 'Parent')->set();
$page2 = Page::make('blip', 'Blip','parent')->set();
```
In fact the title are already displayed in my panel but i can't access of them.. that redirected me in the "customize" (theme) page.

What i forgot? Do i need to add a route (in routes.php) to my option?

Thanks,

N_S

**Comment by jlambe** [comment]
Hmm indeed I've just tested with latest version of the framework and WordPress and it doesn't work as usual. In my case, the child page is displayed twice.
Your code is correct and it's strange that you're redirected to the customizer.
I won't have time to debug this issue this week, I'm sorry. You can still use core functions to build your pages or perhaps use a dedicated plugin so you can still produce your project.

As a last try, did you try to put extra properties in the set method like "capability", "position", "tabs" ? See if it affects the behavior of each page?

I'll keep this issue open in order to fix for the next release. Thanks for reporting this and sorry for the inconvenience.

**Comment by Neidra** [comment]
I investigate a bit this morning. So when you set 
`['tabs' => false]`
I have the same problem than you, in the sub-menus we display also the parent page. 
So the error is created when we set tabs in parent or son page (or both).

**Comment by jlambe** [comment]
Solved for release 1.3.0.
